API-CHANGELOG.md

```diff
@@ -85,10 +85,11 @@ The `network_id` field was added in the `server_info` response in version 1.5.0
 
 ## XRP Ledger server version 2.4.0
 
-As of 2025-01-23, version 2.4.0 is in development. You can use a pre-release version by building from source or [using the `nightly` package](https://xrpl.org/docs/infrastructure/installation/install-rippled-on-ubuntu).
+As of 2025-01-27, version 2.4.0 is in development. You can use a pre-release version by building from source or [using the `nightly` package](https://xrpl.org/docs/infrastructure/installation/install-rippled-on-ubuntu).
 
 ### Additions and bugfixes in 2.4.0
 
+- `simulate`: A new RPC that executes a dry run of a transaction submission without actually submitting it to the network for inclusion.
```

Comment:
I suggest to shorten this line and add a link (possible under the phrase "dry run of a transaction") to https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0069d-simulate#2-rpc-simulate, something like this

```suggestion
- `simulate`: A new RPC that executes a [dry run of a transaction submission](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0069d-simulate#2-rpc-simulate)
```

Reply:
Would also add a point re. fixed fee auto-fill in `submit` , sign-and-submit mode. Do not feel strongly about it.

Reply:
Fixed both in a744645f7d2e7142512791a6bb2abaaff26bd62e